microsoft-edge/visual-studio-code/debugger-for-edge.md

```diff
@@ -1,17 +1,25 @@
 ---
-description: How to debug Microsoft Edge from Visual Studio Code
-title: Debug Microsoft Edge from Visual Studio Code
+description: You can debug Microsoft Edge from Visual Studio Code using the Debugger Extension
+title: Debug Microsoft Edge in Visual Studio Code
 author: MSEdgeTeam
 ms.author: msedgedevrel
-ms.date: 07/26/2021
+ms.date: 11/08/2021
 ms.topic: article
 ms.prod: microsoft-edge
 keywords: microsoft edge, web development, f12 tools, devtools, vs code, visual studio code, debugger
 ---
-# Debug Microsoft Edge from Visual Studio Code
+# Debug Microsoft Edge in Visual Studio Code
 
-[Visual Studio Code](https://code.visualstudio.com) includes a built-in debugger for Microsoft Edge which can launch the browser or attach to an already running browser.
+[Visual Studio Code](https://code.visualstudio.com) includes a built-in debugger for Microsoft Edge, which can launch the browser or attach to an already running browser.
 
-This built-in debugger lets you debug your front-end JavaScript code line-by-line and see `console.log()` statements directly from [Visual Studio Code](https://code.visualstudio.com).
+This built-in debugger lets you debug your front-end JavaScript code line-by-line and see `console.log()` statements directly from Visual Studio Code.
 
-For more information, navigate to [Browser debugging in VS Code](https://code.visualstudio.com/docs/nodejs/browser-debugging).
+For more information, see [Browser debugging in VS Code](https://code.visualstudio.com/docs/nodejs/browser-debugging) and [Browser debugging with Microsoft Edge DevTools integration in Visual Studio Code](/microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension#browser-debugging-with-microsoft-edge-devtools-integration-in-visual-studio-code).
```

Comment:
The second link here is really about the DevTools extension in VS Code. It does talk a tiny bit about debugging JS from VS Code, but then goes on to talk about how you can open DevTools in VS Code by installing the DevTools extension.
So I'm not sure it actually helps users learn more information here.

I still think we need a link to it here. But maybe rephrase to something like:

```
For more information, see [Browser debugging in VS Code](link 1).

You can also inspect HTML and CSS using DevTools from within VS Code. To learn more, see [Browser debugging with Microsoft Edge Devtools integration in VS Code](link 2).
```

Reply:
changed